src/Files.App/Utils/Shell/Win32API.cs

```diff
@@ -217,14 +217,6 @@ public static string ExtractStringFromDLL(string file, int number)
 			}
 		}
 
-		private class IconAndOverlayCacheEntry
-		{
-			public byte[]? Icon { get; set; }
-
-			public byte[]? Overlay { get; set; }
-		}
```

Comment:
We already have the cache in ItemViewModel, is there any benefit in keeping this around?